canopen/objectdictionary/eds.py

```diff
@@ -274,8 +274,10 @@ def build_variable(eds, section, node_id, index, subindex=0):
             logger.warning("%s has an unknown or unsupported data type (%X)", name, var.data_type)
             # Assume DOMAIN to force application to interpret the byte data
             var.data_type = datatypes.DOMAIN
-
-    var.pdo_mappable = bool(int(eds.get(section, "PDOMapping", fallback="0"), 0))
+    
+    pdo_mappable_string = eds.get(section, "PDOMapping", fallback="0")
+    if pdo_mappable_string != "":
+        var.pdo_mappable = bool(int(pdo_mappable_string, 0))
```

Comment:
I guess it is still possible to have invalid input.

Maybe something like this?

```python
var.pdo_mappable = eds.get(section, "PDOMapping", fallback="0") not in ("", "0")
```

Reply:
I think we want the invalid input to throw an exception so real errors are caught, though I guess it's a question whether `PDOMapping=` is valid input or not? 

I no longer have access to the EDS files that were causing the issue so I don't have anything to test against.
If I have time I'll try and poke around to see if I can find any other files that exhibit this issue.

it does make me wonder if the handling of invalid input on the other entries should also throw an error?